src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +564,15 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        if (self::$constantSupport) {
+                            return @constant(substr($scalar, 11));
```

Comment:
@stof there is a hidden conversation on a diff above between me and @xabbuh where we argue about this. It returns `null` but there is a warning logged as silenced error thanks to #16760.
Isn't that enough? Maybe we could also silence it in master for the `XmlLoader`?